### ISSUE TITLE
Private Gitlab server

### ISSUE THREAD

**Issue by pawelkasperek** [gmasterscm/ama#14]
Hi,

I have private Gitlab Enterprise instance. The functionality and API is the same. Would do you like add possibility to set URL to private Gitlab server ?

**Comment by psantosl** [comment]
Hey, I think you can check this one:

https://github.com/gmasterscm/ama/issues/11

**Comment by psantosl** [comment]
Yes, we have it in mind. I think it has been requested a few times already :-)
I believe you can still use gmaster with it, but you can't browse remote repos, correct?

**Comment by pawelkasperek** [comment]
Yes, I started use gmaster because it has semantic merge and it's faster than SourceTree 👍  .  Of course I can add repository, but I need non stop put authenticatin password. When I can add private Gitlab server (ex. https://myprivate.gitlab.me) that I can use the same as Github, Bitbucket or publicm Gitlab.com server. 
I think that this functionality isn't now important.

**Comment by psantosl** [comment]
Thanks for your understanding and for using gmaster!! Really encouraging that you find it useful even when some feature is missing.
We will add gitlab server support soon! 👍
